packages/server/postgres/queries/getTeamByTeamId.ts

```diff
@@ -0,0 +1,10 @@
+import {getTeamByTeamIdQuery, IGetTeamByTeamIdQueryResult} from './generated/getTeamByTeamIdQuery'
+import catchAndLog from '../utils/catchAndLog'
+import getPg from '../getPg'
+
+const getTeamByTeamId = async (teamId: string): Promise<IGetTeamByTeamIdQueryResult> => {
+  const teams = await catchAndLog(() => getTeamByTeamIdQuery.run({id: teamId}, getPg()))
+  return teams![0]
```

Comment:
curious why add the bang operator?  what happens if `catchAndLog` returns null? should we add `teams === null ? [] : teams` before returning the first value, similarly to `packages/server/postgres/queries/getTeamsByOrgId.ts`?

Reply:
This method is expected to return a single team object, isn't it? I did have difficulty in handling the nullify here. Any better idea?

Reply:
i think we could do 

```
teams === null ? [] : teams
return teams[0]
```

if the query returns null, then this wrapper will return undefined instead of raising an error

Reply:
let's nuke this file & just use `getTeamsById`